posts/2016-09-20-IDA2016-notes.md

```diff
@@ -0,0 +1,129 @@
+
+From the perspective of Dat, the costs were the most interesting part. For many of the data repositories hosting and bandwidth covers 1/4th to 1/3rd of the costs. We hope that Dat will be able to better distribute or reduce a lot of those costs.
+
+Repositories that offer *active* dataset curation also had a high cost for the curators, a relatively new profession, at around 1/3 of the total costs.
+
+Finally, the last big chunk of spending was on software development and general research and development. This is not surprising, software is expensive. But it was surprising to see how many repositories were replicating the same costs developing similar software.
+
+### [Managing Science Data: A Federal Agency’s Perspective](http://www.scidatacon.org/2016/sessions/10/)
+
+This session featured the challenges faced by federal agencies around data management. Recent federal mandates have started to require open data and agencies are moving towards that goal. The papers from the session are worth a read for the different perspective, compared to academic researchers.
+
+The [USGS data management plan](https://www2.usgs.gov/datamanagement/why.php) is a well thought out plan on each stage of the data lifecycle, how the USGS is approaching it, and summaries of the federal requirements around data.
+
+## International Data Forum
+
+The highlight of the forum was the panel on "Open Data as a Public Good and the Responsibilities of Scientists".
+
+The three presentation slides are available online:
+
+* Myron P. Gutmann, University of Colorado Boulder: [“Balancing Scientific Responsibility and the Public Good”](https://docs.google.com/viewer?url=http%3A%2F%2Fwww.internationaldataweek.org%2Fsites%2Fdefault%2Ffiles%2FMyron%2520P.%2520Gutmann.pptx%23overlay-context%3DInternational-data-forum)
+* Victoria Stodden, University of Illinois at Urbana-Champaign: [“Scientific Transparency, Integrity and Reproducibility”](http://www.internationaldataweek.org/sites/default/files/Victoria%20Stodden.pdf#overlay-context=International-data-forum)
+* Takashi Onishi, President of Science Council of Japan (SCJ): [“Open Data as Game Changer for Natural Disaster Prevention”](https://docs.google.com/viewer?url=http%3A%2F%2Fwww.internationaldataweek.org%2Fsites%2Fdefault%2Ffiles%2FTakashi%2520Onishi.pptx%23overlay-context%3DInternational-data-forum)
+
+The presenters made clear that is a strong case to made for why open data is good for the public, especially in cases like natural disaster prevention. However, it is unreasonable to say all research data must be totally open. There are privacy concerns, IP issues, and technological hurdles. Making specific recommendations, such as around reproducibility of published results, can go a long way in supporting both the public good and research needs.
```

Comment:
Also: first sentence reads like there's a good argument for making an argument for something. To me that sounds slightly odd haha